### PR TITLE
Update typo in 6.11 Blocks of specs

### PR DESCRIPTION
**Pull request by liang3zy22**

### CODE REVIEW

**Comment by liang3zy22** [comment]
Already signed the CLA.

File: spec/06-expressions.md [hunk 1]
```diff
@@ -599,7 +599,7 @@ Evaluation of the block entails evaluation of its
 statement sequence, followed by an evaluation of the final expression
 ´e´, which defines the result of the block.
 
-A block expression `{´c_1´; ´\ldots´; ´c_n´; ´}` where ´s_1 , \ldots , s_n´ are
+A block expression `{´s_1´; ´\ldots´; ´s_n´; }` where ´s_1 , \ldots , s_n´ are
```
Comment (som-snytt) [inline spec/06-expressions.md]: The meaning is: if the `s_n` of the block are case clauses `c_n`. So `c` is intended. The extra quote must be a typo, but the final semicolon is also a typo.  Then the `s_1` should just be `c_1` etc. Note that this is not grammar syntax and the semicolons are also only for clarity.

**Review by som-snytt — CHANGES_REQUESTED** [review]

File: spec/06-expressions.md (hunk 1 above)
Comment (liang3zy22) [reply spec/06-expressions.md]: Oh, I got it. I didn't knew its actual meaning so made the first commit. I added a new commit, just removed the typo.

**Comment by liang3zy22** [comment]
I created two commits. Is it redundant? I can close this pull request if redundant commit.

**Comment by liang3zy22** [comment]
I  merged the two commit into one. Hope it follows the procedure.

**Review by SethTisue — APPROVED** [review]
Thank you! We'll merge it after 2.13.9 is out.

**Pull request merged**